File: zgadywanie.py
import random

print("Hello")

random_number =random.randint(0,  100)
i = 0
while True:
    i +=1
    liczba = int(input("Wylosowalem liczbe od 0 do 100: zgadnij liczbe>>>     "))
    if liczba < random_number:
        print("Moja liczba jest wieksza")
    if liczba > random_number:
        print("Moja liczba jest mniejsza")
    if liczba == random_number:
        print("Zgadles liczbe w", i, "probie!")
        break

File: zgadywanie.cpp
#include<iostream>
#include <cstdlib>
#include <ctime>


using namespace std;

int main(){
    int liczba;
    int losowa;
    int i =0;
    srand(time(0));
    losowa = rand() % 101;

    cout << "Wylosowalem liczbe od 0 do 100\n";

    while(true){
        i++;
        cout << "sproboj zgadnac>>    ";
        cin >> liczba;
        cout << "\n";
        if(liczba > losowa){
            cout<< "wylosowalem mniejsza \n";
        }
        else if(liczba < losowa){
            cout << "wylosowalem wieksza \n";
        }
        else if(liczba == losowa){
            cout << "brawo wygrales w " << i << "probie \n";
            break;
        }
    }

    return 0;
}